module/apmfasthttp/context.go

```diff
@@ -85,9 +87,7 @@ func setResponseContext(ctx *fasthttp.RequestCtx, tx *apm.Transaction, bc *apm.B
 	tx.Context.SetHTTPResponseHeaders(headers)
 	tx.Context.SetHTTPStatusCode(statusCode)
 
-	if bc != nil {
-		bc.Discard()
-	}
+	bc.Discard()
```

Comment:
Now that setResponseContext is called from within the handler, I'm not sure if this is too early or not. To be on the safe side, I think we should probably remove this and just call from the closer.